### PR TITLE
[jsfm] Add validation when binding events

### PR DESCRIPTION
**Pull request by Hanks10100**
Also added corresponding unit test case.

### CODE REVIEW

File: html5/shared/utils.js [hunk 1]
```diff
@@ -76,6 +76,11 @@ function hasOwn (obj, key) {
  */
 
 function bind (fn, ctx) {
+  if (typeof fn !== 'function') {
```
Comment (Jinjiang) [inline html5/shared/utils.js]: Well, the function `bind` is a shared util method. I suggestion keep it simple and quick. If you want to ignore errors when function binding it's better do the check in some particular situation code (for example event binding).
Thanks.

Comment (Hanks10100) [inline html5/shared/utils.js]: OK. There is already a type check in directive `bindEvents`, I just change the `debug` info to an `error`.

File: html5/frameworks/legacy/vm/directive.js [hunk 2]
```diff
@@ -282,7 +282,7 @@ function bindEvents (vm, el, events) {
       handler = vm[handler]
       /* istanbul ignore if */
       if (!handler) {
-        console.debug(`[JS Framework] The method "${handler}" is not defined.`)
+        console.error(`[JS Framework] The event handler "${handler}" is not defined.`)
```
Comment (Jinjiang) [inline html5/frameworks/legacy/vm/directive.js]: Just `console.warn` is OK.

Comment (Hanks10100) [inline html5/frameworks/legacy/vm/directive.js]: OK, changed.

**Pull request merged**